test/suites/test_push.py

```diff
@@ -0,0 +1,449 @@
+import sys
+import unittest
+import warnings
+import tarantool
+from tarantool.error import (
+    NetworkError,
+    NetworkWarning,
+)
+from .lib.tarantool_server import TarantoolServer
+from .lib.skip import skip_or_run_push_test_via_conn_pool
+
+
+def create_server():
+    srv = TarantoolServer()
+    srv.script = 'test/suites/box.lua'
+    srv.start()
+    srv.admin("box.schema.user.create('test', {password = 'test', " +
+              "if_not_exists = true})")
+    srv.admin("box.schema.user.grant('test', 'read,write,execute', 'universe')")
+
+    # Create server_function (for testing purposes).
+    srv.admin("""
+    function server_function()
+        x = {0,0}
+        while x[1] < 3 do
+            x[1] = x[1] + 1
+            box.session.push(x)
+        end
+        return x
+    end
+    """)
+
+    # Create tester space and on_replace trigger (for testing purposes).
+    srv.admin("""
+    box.schema.create_space(
+        'tester', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+        }
+    })
+    """)
+    srv.admin("""
+    box.space.tester:create_index(
+        'primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        }
+    })
+    """)
+    srv.admin("""
+    box.space.tester:create_index(
+        'primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        }
+    })
+    """)
+    srv.admin("""
+    function on_replace_callback()
+        x = {0,0}
+        while x[1] < 300 do
+            x[1] = x[1] + 100
+            box.session.push(x)
+        end
+    end
+    """)
+    srv.admin("""
+    box.space.tester:on_replace(
+        on_replace_callback
+    )
+    """)
+
+    return srv
+
+
+# Callback for on_push arg (for testing purposes).
+def push_callback(data, on_push_ctx=[]):
+                data[0][1] = data[0][1] + 1
+                on_push_ctx.append(data)
+
+
+@unittest.skipIf(sys.platform.startswith("win"),
```

Comment:
Mesh=ConnectionMesh tests, and I don't think they are

Reply:
Ouch, fixed

Reply:
Why push tests on windows platform are not supported? I don't see any restrictions from both Tarantool and connector side.

Reply:
Done, deleted this skip